rootfs/scheduler/__init__.py

```diff
@@ -176,13 +177,14 @@ def _get_deploy_batches(self, steps, desired):
 
         return batches
 
-    def _update_application_service(self, namespace, name, app_type, port, routable=False):
+    def _update_application_service(self, namespace, name, app_type, port, routable=False, maintenance=False):  # noqa
```

Comment:
perhaps we should rewrite this to read for an `annotations` kwarg and attach that to the service metadata instead of making this an N-length method, where N is the number of supported annotations in the router.